test/cljs/status_im/test/protocol/core.cljs

```diff
@@ -3,102 +3,103 @@
                                       async use-fixtures]]
             [cljs.nodejs :as nodejs]
             [cljs.core.async :as async]
-            [status-im.protocol.web3.utils :as web3.utils]
+            ;;[status-im.protocol.web3.utils :as web3.utils]
```

Comment:
lets just remove tests under `sstatus-im.test.protocol` and `protocol` cljs build from project.clj